compiler/rustc_feature/src/removed.rs

```diff
@@ -101,6 +101,10 @@ declare_features! (
      Some("never properly implemented; requires significant design work"), 127655),
     /// Allows deriving traits as per `SmartPointer` specification
     (removed, derive_smart_pointer, "1.84.0", Some(123430), Some("replaced by `CoercePointee`"), 131284),
+    /// Tells rustdoc to automatically generate `#[doc(cfg(...))]`.
+    (removed, doc_auto_cfg, "1.58.0", Some(43781), Some("merged into `doc_cfg`"), 138907),
```

Comment:
Is this the correct version? It generates an error message like:

```
error[E0557]: feature has been removed
 --> zeroize/src/lib.rs:2:29
  |
2 | #![cfg_attr(docsrs, feature(doc_auto_cfg))]
  |                             ^^^^^^^^^^^^ feature has been removed
  |
  = note: removed in 1.58.0; see <https://github.com/rust-lang/rust/pull/138907> for more information
  = note: merged into `doc_cfg`
```

Seems like it was added in 1.58.0, not removed in 1.58.0...

Reply:
Already fixed by #147123 which is gonna be in the next nightly.